src/qt/test/test_main.cpp

```diff
@@ -63,7 +63,7 @@ int main(int argc, char *argv[])
     #if defined(WIN32)
         _putenv_s("QT_QPA_PLATFORM", "minimal");
```

Comment:
Would suggest adding `if (getenv("QT_QPA_PLATFORM") == nullptr)` so the new instructions for setting `QT_QPA_PLATFORM` work on windows. Or, if you'd prefer to not change behavior in a documentation-only PR, you could add a `// TODO: Fix this to not overwrite QT_QPA_PLATFORM if it is set externally` comment

Reply:
Thx, added a commit and changed the pull request type to "qa"